utilities/resourceRemoval/Remove-NetAppAccount.ps1

```diff
@@ -0,0 +1,64 @@
+﻿function Remove-NetAppAccount {
+
+    [Cmdletbinding(SupportsShouldProcess)]
+    param(
+        [Parameter(Mandatory = $false)]
+        [string] $moduleName = 'netAppAccounts',
+
+        [Parameter(Mandatory = $false)]
+        [string] $ResourceGroupName = 'validation-rg'
+    )
+
+    $retryCount = 1
+    $retryLimit = 40
+    $retryWaitInSeconds = 15
```

Comment:
Suggestion: Put these as parameters with default values.
Wont do it in the suggestion, but need to also fix casing on params and documentation on functions.

Not for now: I think the retry mechanism should be outside of this script. I'm not sure it should not have to be up to each script to handle retries. Thinking more the script should have logic to handle a removal, if it throws an error it returns to caller, and caller handles the retry by calling the script again. But there are probably reasons to have retries in these scripts too I can imagine.

```suggestion
    [Cmdletbinding(SupportsShouldProcess)]
    param(
        [Parameter(Mandatory = $false)]
        [string] $moduleName = 'netAppAccounts',

        [Parameter(Mandatory = $false)]
        [string] $ResourceGroupName = 'validation-rg',
        
        [Parameter(Mandatory = $false)]
        [int] $retryLimit = 40,
        
        [Parameter(Mandatory = $false)]
        [int] $retryWaitInSeconds = 15
    )

    $retryCount = 1
```

Reply:
Agreed. I'm currently working on that & aligning the approaches (for resources) too